src/background/Wallet/model/types.ts

```diff
@@ -102,9 +103,15 @@ export interface WalletRecordVersion4
   feed: WalletFeed;
 }
 
-export interface WalletRecord extends Omit<WalletRecordVersion4, 'version'> {
+export interface WalletRecordVersion5
+  extends Omit<WalletRecordVersion4, 'version'> {
+  /** This version introduces normalized addresses in "permissions", see ./versions.ts */
   version: 5;
-  /** This version introduces normalized addresses in "permissions" */
+}
+
+export interface WalletRecord extends Omit<WalletRecordVersion5, 'version'> {
+  version: 6;
+  activityRecord: ActivityRecord;
```

Comment:
I wasn't sure if this needed a version upgrade. We could make this as optional and keep version 5. But it seems more correct to upgrade